src/market-programs.ts

```diff
@@ -822,6 +822,21 @@ export const marketPrograms: MarketRewardProgramArgs[] = [
     },
     chainId: ChainId.BASE,
   },
+  // pythUSDC - weETH borrow rewards 25,0000 PYTH 09/6-12/06 00:00 UTC
+  {
+    start: 1726520400n,
+    end: 1734382800n,
+    fundsSender: "0xAaB35e8702D783428EE2e1929CDB99e58093C3E2",
+    urdAddress: "0x5400dBb270c956E8985184335A1C62AcA6Ce1333",
+    tokenAddress: "0x4c5d8A75F3762c1561D96f177694f67378705E98",
+    marketId: "0xa4e2843486610e6851f4e0a8fcdee819958598c71c7e99b0315904ccf162ddc3",
+    rewardAmount: {
+      supply: 0n,
+      borrow: parseUnits("50000", 18),
```

Comment:
```suggestion
      borrow: parseUnits("50000", 6),
```

PYTH is in 6 decimals not in 18 (same for other programs).